change/@ni-nimble-tokens-b264b0ee-0471-4955-99c5-10e8b68dff74.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

This is adding new APIs so it should be minor rather than patch
```suggestion
  "type": "minor",
```

---

Updated

---

Looks like you updated `dependentChangeType` but not `type`. The bump for the nimble-tokens package is controlled by `type` so it should definitely be `minor` because that's where the new APIs are being added. The bump for `nimble-components` package is controlled by `dependentChangeType`. It doesn't re-export these APIs and we're not adding any new ones, so it should be `patch`.